modules/users/client/controllers/settings/change-password.client.controller.js

```diff
@@ -33,11 +33,13 @@
       // If successful show success message and clear form
       $scope.$broadcast('show-errors-reset', 'vm.passwordForm');
       vm.success = true;
```

Comment:
Im not at a computer right now so i cant give examples, or links..

In some of our Karma tests, you'll see we have a Spy on some of the angular controller services. You can add some in these tests that ensure the Notification service was called; I forget the syntax. If a simple spy doesnt work, you can look into Sinon's Stub feature.